index.html

```diff
@@ -56,5 +56,9 @@
     </div>
   </div>
   <!-- built files will be auto injected -->
+  <noscript>
+    <h3>We're sorry but this site requires a browser that supports JavaScript.</h3>
+    <strong>Your browser either does not support JavaScript, or it has JavaScript support disabled. If you want to correctly view this page, please upgrade your browser or enable JavaScript support.</strong>
+  </noscript>
```

Comment:
I think we need to do it a bit better. Add some styles, info about browsers that we support. Look at this an example: https://github.com/tokend/client-scaffold/blob/master/src/vue/common/SupportedBrowsers.vue